package.json

```diff
@@ -14,7 +14,10 @@
     "webpack-dev-server": "^4.9.0"
   },
   "dependencies": {
+    "@codemirror/lang-json": "^6.0.1",
+    "@codemirror/theme-one-dark": "^6.1.0",
     "@datadog/browser-logs": "^4.13.0",
+    "codemirror": "^6.0.1",
```

Comment:
Poee trek jij dit wel? Al die dependencies 😝

Reply:
Met moeite 🙈 Gelukkig is het eigenlijk één dependency met een paar plugins. En Codemirror is redelijk spaarzaam met dependen op andere dingen.